tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "noEmitOnError": true,
     "sourceMap": true,
     "outDir": "dist",
-    "rootDir": "./",
+    "rootDir": "src",
```

Comment:
should perhaps have been a separete commit (?)

Reply:
This is fine.